test/trackler/problem_test.rb

```diff
@@ -82,6 +82,10 @@ def test_track_with_no_canonical_data
     assert_equal nil, problem.canonical_data_url
   end
 
+  def test_problem_with_no_metadata_yml
+    refute Trackler::Problem.new('no-metadata', FIXTURE_PATH).exists?
+  end
```

Comment:
Something should happen if there is no metadata file. What do you want this to be?
I would rather it is a failure with a useful error message rather than just blowing up.